### PR TITLE
[3.x] Adds Laravel 11 support

### PR DESCRIPTION
**Pull request by nunomaduro**
This pull request is a work in progress, and it adds Laravel 11 support.

### CODE REVIEW

**Comment by calebporzio** [comment]
Thanks, looking forward to it

File: composer.json [hunk 1]
```diff
@@ -10,22 +10,28 @@
...
     "calebporzio/sushi": "^2.1",
-    "laravel/prompts": "^0.1.6"
+    "laravel/prompts": "^0.1.6|dev-main"
   },
+  "repositories": [
+    {
+      "type": "git",
+      "url": "https://github.com/laravel-labs/php-webdriver"
+    }
+  ],
```
Comment (crynobone) [inline composer.json:34]:
```suggestion
```

Comment (crynobone) [reply composer.json:34]: Should be safe to remove now.

File: composer.json [hunk 2]
```diff
@@ -10,22 +10,28 @@
...
-    "laravel/framework": "^10.0",
-    "orchestra/testbench": "^8.0",
-    "orchestra/testbench-dusk": "^8.0",
+    "phpunit/phpunit": "^9.0|^10.0",
+    "laravel/framework": "^10.0|^11.0",
+    "orchestra/testbench": "^8.0|^9.0",
+    "orchestra/testbench-dusk": "^8.0|^9.0",
     "calebporzio/sushi": "^2.1",
-    "laravel/prompts": "^0.1.6"
+    "laravel/prompts": "^0.1.6|dev-main"
```
Comment (crynobone) [inline composer.json:27]:
```suggestion
    "laravel/prompts": "^0.1.6"
```

Symfony 7 support is available via `0.1.12`

File: composer.json [hunk 3]
```diff
@@ -10,22 +10,28 @@
...
   },
   "require-dev": {
-    "psy/psysh": "@stable",
+    "psy/psysh": "^0.11.22|^0.12",
     "mockery/mockery": "^1.3.1",
-    "phpunit/phpunit": "^9.0",
-    "laravel/framework": "^10.0",
-    "orchestra/testbench": "^8.0",
-    "orchestra/testbench-dusk": "^8.0",
+    "phpunit/phpunit": "^9.0|^10.0",
```
Comment (crynobone) [inline composer.json:22]: Might be best to migrate to PHPUnit 10 since both Laravel 10 and 11 supports it. PHPUnit 9 isn't supported in Laravel 11.

**Pull request merged**